product-scenarios/4-single-sign-on/4.4-application-management/4.4.1-soap-apis/src/test/java/org/wso2/identity/scenarios/test/ApplicationManagementTestCase.java

```diff
@@ -89,7 +91,7 @@ public void testInit() throws Exception {
     @AfterClass(alwaysRun = true)
     public void atEnd() throws Exception {
         applicationManagementServiceClient = null;
-        userMgtClient.deleteRole(IDP_ROLE_1);
+//        userMgtClient.deleteRole(IDP_ROLE_1);
```

Comment:
why commented here

Reply:
We are adding new role here. Re-using internal(admin role)

Reply:
Fixed with https://github.com/wso2/product-is/pull/14317/commits/1b4ab737ecfa0d7092d2c697a13b8ad39d9ad538